### DISCUSSION TITLE
Having a problem? Use Issues!

### DISCUSSION THREAD

**Discussion by rosie-at-pieces** [pieces-app/support#2] [Announcements]
We encourage community-driven support, and the Pieces team is very active on our issues page and will respond to you as fast as possible. 

Your issue might have been seen before, so check out the FAQs or do a search on your issue before you create a new one!

**Comment by patelishwar22** [comment]
App installation failed with error message: Appinstaller operation failed with error code 0x80070422. Detail: The service cannot be started, either because it is disabled or because it has no enabled devices associated with it. (0x80070422)

I AM GETTING THIS ERROR WHILE INSTALLING THIS APP. NEED HELP!

**Reply by rosie-at-pieces** [reply]
hi @ishupatel07 please create an issue in the [Issue page](https://github.com/pieces-app/support/issues)

**Reply by patelishwar22** [reply]
How to create that I didn't see creat issue filed anywhere

**Comment by 0gopi0** [comment]
i am using amr 64 ubuntu how to install pices on it

**Reply by rosie-at-pieces** [reply]
Hey @0gopi0 check out our [documentation](https://docs.pieces.app/installation-getting-started/linux) on how to install on Linux!

**Comment by anupchauhanak6** [comment]
![Screenshot 2024-07-19 001653](https://github.com/user-attachments/assets/21795bea-1e85-4488-9134-60b4eeaa10c5)

**Reply by anupchauhanak6** [reply]
![Uploading Screenshot 2024-07-19 005656.png…]()

**Reply by rosie-at-pieces** [reply]
Hi @aak3645 please open an issue for this on https://github.com/pieces-app/support/issues!

**Comment by KTrager** [comment]
PiecesOS failed to download. Haven been trying this again and again, but can't get past this. My connection works otherwise I'll not be able to write this.
How to get around this?

<img width="1436" height="1232" alt="CleanShot 2026-02-02 at 12 53 37@2x" src="https://github.com/user-attachments/assets/dfcd329a-8709-47e4-b884-74f6da90a8df" />

**Comment by tomassales-collab** [comment]
The app has been installed with the cloud option enabled by default. It causes security problems
